services/load/service.go

```diff
@@ -335,7 +335,7 @@ func (s *Service) loadTask(f string) error {
 	}
 	s.tasks[id] = true
 
-	return nil
+	return err
```

Comment:
Won't err always be nil here?

Reply:
The idea was to return err from the defer, but I removed that.